docs/docs/faq.mdx

```diff
@@ -63,6 +63,16 @@ The recommended way to run an A/B test is by using Feature Flags and our SDKs.
    }
    ```
 
+## What is the best way to redirect users to a URL based on their experiment variation?
+
+Experimenters often find themselves wanting to show one URL to some users and another URL to other users. This is not always trivial, because waiting to look up a user's variation, and then rendering a page for them, could cause flickering in the user experience and even bias, if you are only delaying page load for one variation.
+
+In general, we recommend three approaches, listed from least to most risky with respect to user experience and bias:
+
+1. Conditionally render of page A vs. page B using the GrowthBook SDK within backend controller logic. This is fast, has no bias, and there is no flickering/jumpy redirection.
+2. Conditionally render on the front-end page A vs. page B ing the GrowthBook JS SDK. Almost as good as option 1, although there is a slight loading delay. This can also be done with no bias if both A and B incur the same page load penalty. Preferably you can implement this without any flickering for users, but there will be some slight loading delay.
+3. Frontend redirection using JS snippets in the visual editor using `window.location.href="mytarget.html"`. Problematic but likely good enough if done carefully. Incurs noticeable flickering before/after redirection. Ideally the redirection should happen from a neutral / empty-ish page for both the A and B treatments as to not introduce bias (e.g. avoiding the case where page A just loads faster due to the experiment config so people like it more).
```

Comment:
There are a few run on sentences here and a typo or two.  I ran it through Chat GPT saying: "How would you rephrase this for clarity" and I think what it came up with reads easier.  I did edit it a bit then in places where it didn't quite keep the same meaning:  

> Experimenters often face the challenge of displaying different URLs to different users. However, achieving this isn't always straightforward. Delaying the retrieval of a user's variation and rendering a page accordingly can lead to undesirable effects like flickering in the user interface and potential bias. This bias can occur if the delay in page load only affects one of the variations.
> 
> In general, we suggest three approaches, ranked from the least risky to the most risky in terms of user experience and potential bias:
> 
> 1. Render page A or page B conditionally using the GrowthBook SDK within the backend controller logic or middleware layer. This method is quick, unbiased, and avoids any flickering or sudden redirection.
> 2. Render page A or page B conditionally using the GrowthBook JS SDK on the front-end. This option is nearly as good as the first one, and while you might be able to implement this without any flickering for users, it might have a slight loading delay. It can be unbiased if both pages A and B result in the same loading delay. 
> 3. Perform front-end redirection using JS code snippets in the visual editor with the command `window.location.href="mytarget.html"`. This approach is problematic but can be acceptable if executed carefully. It introduces noticeable flickering before and after redirection. For fairness, it's best to initiate the redirection from a neutral or minimally loaded page for both variations A and B. This helps avoid introducing bias—for instance, preventing a scenario where page A appears more favorable simply because it loaded faster due to experiment configurations.

Reply:
Didn't love a lot of the language it used here, but it provided a useful foil to rewrite for clarity. Will push and land, thanks!

Reply:
The list of 1 2 3 is a lot clearer, but I think you probably touched it up quite a bit :)